s/data.🍇

```diff
@@ -13,7 +13,12 @@
     🚜 data 0 memory 0 count❗️
   🍉
 
-  📗 Returns the memory region as a 🧠 📗
+  📗
+    Returns the 🧠 storing the value of this 📇. No copy is performed.
+
+    Do not free the 🧠 manually. Do not use 🧠 after the 📇 goes out of scope.
```

Comment:
Can you please change this line to something like “only read from this”? 🧠 is reference counted so neither can it be freed nor can there be use-after-free.